client/src/pages/Project.js

```diff
@@ -0,0 +1,229 @@
+import React from "react";
+
+import {
+  Typography,
+  Container,
+  Box,
+  Avatar,
+  Button,
+  Card,
+  CardMedia,
+  Divider,
+  Tabs,
+  Tab,
+  List,
+  ListItem,
+} from "@material-ui/core";
+
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+
+import projectMain from "../staticImages/projMain.png";
+import projectInfo from "../staticImages/projInfo.png";
+import profpic1 from "../staticImages/profpic1.png";
+
+import NavBar from "../components/Navbar";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    color: "black",
+  },
+  userDividerTop: {
+    marginTop: theme.spacing(3),
+  },
+  userDividerBottom: {
+    marginBottom: theme.spacing(3),
+  },
+}));
+
+const userStatic = {
+  name: "Alexander Faa",
+  location: "New York, NY",
+};
+const projectStatic = {
+  title: "Urban Jungle",
+  subtitle: "Coffee. Community",
+  currentInvested: 29000,
+  wantedInvestement: 40000,
+  equity: 0.1,
+  timeLeft: 44,
+  backers: 50,
+  photos: [projectMain, projectInfo],
+  about: {
+    main_about:
+      "Kopi-luwak cultivar, cup organic cup americano trifecta aged. Iced, frappuccino white ristretto affogato a cappuccino. Iced whipped et  decaffeinated, americano, aromatic so cultivar sugar robusta. Aged decaffeinated, siphon redeye, organic frappuccino froth crema java",
+    additional_info: [
+      "Kopi-luwak cultivar, cup organic cup americano trifecta aged.",
+      "Iced, frappuccino white ristretto affogato a cappuccino",
+      "Iced whipped et black carajillo decaffeinated, americano, aromatic so cultivar sugar robusta",
+    ],
+  },
+};
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        // You want everything in here to have lots of vertical space from each other, ideally without wrapping every single component in a spacing box
+        <Box p={3} textAlign="left">
+          {children}
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    "aria-controls": `simple-tabpanel-${index}`,
+  };
+}
+
+function Header(props) {
+  const proj = props.project;
+  return (
+    <div>
+      <Button variant="outlined" size="large">
+        Crafts
+      </Button>
+      <Typography variant="h2">
+        <Box fontWeight="fontWeightMedium">{proj.title}</Box>
+      </Typography>
+      <Typography color="textSecondary">{proj.subtitle}</Typography>
+    </div>
+  );
+}
+
+function ProjectPanel(props) {
+  const [value, setValue] = React.useState(0);
```

Comment:
It's a bit cleaner to import useState at the top, i.e. `import React, { useState} from "react"`

Reply:
Thanks, change is made